src/Core/Components/TreeView/FluentTreeItem.razor.cs

```diff
@@ -6,6 +6,20 @@ public partial class FluentTreeItem : FluentComponentBase, IDisposable
 {
     private bool _disposed;
 
+    /// <summary>
+    /// Initializes a new instance of the <see cref="FluentTreeItem"/> class.
+    /// </summary>
+    public FluentTreeItem()
+    {
+        Id = Identifier.NewId();
+    }
+
+    /// <summary>
+    /// Gets or sets the list of sub-items to bind to the tree item
+    /// </summary>
+    [Parameter]
+    public IEnumerable<ITreeViewItem>? Items { get; set; }
```

Comment:
What about `Children`, `ChildrenItems`, `SubItems`?

Reply:
What do you mean? About the property name?
- `Children` and `ChildrenItems` are too close to `ChildContent` (I think)
- `SubItems` why not

But why ? :-)

Reply:
Yes for the property name. I think it could clear up some confusion to have an `Items` property in a class named `Item`